### ISSUE TITLE
LDA and QDA

### ISSUE THREAD

**Issue by farnsl01** [DerekBrownCS/STAT518Project#3]
There is code. It is untested.

**Comment by farnsl01** [comment]
LDA and QDA work now! model made with training set, predicts the testing set and creates a confusion matrix. Need to use the validation set.
